tracing-journald/src/lib.rs

```diff
@@ -268,6 +298,60 @@ where
     }
 }
 
+/// The facility argument is used to specify what type of program is logging the
+/// message. This lets the configuration file specify that messages from
+/// different facilities will be handled differently.
+#[derive(Copy, Clone)]
+pub enum SyslogFacility {
```

Comment:
This enum is a significant increase in API surface area, and may be a portability hazard. It'd be safer to take an arbitrary `i32` and leave the problem of identifying the appropriate value to the caller, who could then use libc or other sources themselves. This also makes the feature usable in cases where libc is missing a definition.

Reply:
Some of these facilities are defined by POSIX / Single Unix Specification. However, different versions seem to define different set. I was also hoping journald is Linux-only (at least for now) so the definitions should stay relatively stable. The reason for this enum was to not giving end user the possibility of making mistake with libc definitions and doing something not supported by syslog or journald. But I'm fine with `c_int` if that's what is wanted.

Reply:
I'd prefer to err on the side of a smaller, more stable API.

Reply:
I removed the `SyslogFacility` enum.